src/lib/arch/aarch64/InstructionMetadata.hh

```diff
@@ -84,6 +84,13 @@ struct InstructionMetadata {
   /** The number of explicit operands. */
   uint8_t operandCount;
 
+  /** The current exception state of this instruction. */
+  simeng::arch::aarch64::InstructionException metadataException_ =
```

Comment:
`simeng::arch::aarch64::` is unnecessary given the `namespace` usage

Reply:
Of course, think this has remained after an initial implementation. Removed